packages/embed/src/components/pausedpopover/ListenOnAudiusCTA.jsx

```diff
@@ -23,9 +23,11 @@ const ListenOnAudiusCTA = ({ audiusURL, premiumConditions }) => {
     >
       <Text variant='title' size='l'>
         {isPurchaseable
-          ? `${messages.buy} $${new FixedDecimal(
+          ? `${messages.buy} $${USDC(
               premiumConditions.usdc_purchase.price / 100
-            ).toLocaleString('en-US', { minimumFractionDigits: 2 })}`
+            ).toLocaleString('en-US', {
+              minimumFractionDigits: 2
```

Comment:
still might want the `maximumFractionDigits: 2` (though it's unlikely to be necessary, `USDC` doesn't do it for you (yet))

Reply:
kk